config/config_mgmt.py

```diff
@@ -71,18 +68,24 @@ def __init__(self, source="configDB", debug=False, allowExtraTables=True):
 
         return
 
-    def logInFile(self, header="", obj=None, json=False):
+    def logInFile(self, msg=None, logLevel='INFO'):
```

Comment:
why can't we get rid of this function all together and use syslog.syslog() function everywhere? why I am saying so is, we don't want to look into the map for every log message. Isn't it?

Reply:
A wrapper function is always good, if later we need to make change in single function to append prefix\suffix to each function. This is followed similar way in each python app in sonic till now. But since we do not have such requirement, I am fine even to remove wrapper function.